docs/devops.md

```diff
@@ -4,110 +4,185 @@ title: DevOps
+
+![](https://user-images.githubusercontent.com/5632966/105532713-d0ae1080-5cb0-11eb-8d8b-5d3751b7df37.png)
 
-<figure class="video_container">
-  <iframe width="100%" height="315" src="https://www.youtube.com/embed/53X-HAw7BbA" frameborder="0" allowfullscreen="true"> </iframe>
-</figure>
 
 * * *
 
-### Introducción a **Terraform** 
+
+## Infraestructura como código (IAC)
+
+Es el proceso de gestión y aprovisionamiento de centros de datos informáticos a través de archivos de definición legibles por máquina, en lugar de configuración de hardware físico o herramientas de configuración interactiva.
 
 <figure class="video_container">
-  <iframe width="100%" height="315" src="https://www.youtube.com/embed/cpxKbf51ccU" frameborder="0" allowfullscreen="true"> </iframe>
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/53X-HAw7BbA" frameborder="0" allowfullscreen="true">
+  </iframe>
 </figure>
 
 * * *
 
-### Introducción a **HashiCorp Terraform**
+### Introducción a Terraform 
+
+Terraform es un software de infraestructura como código desarrollado por HashiCorp. Permite a los usuarios definir y configurar la infraestructura de un centro de datos en un lenguaje de alto nivel.
 
 <figure class="video_container">
-  <iframe width="100%" height="315" src="https://www.youtube.com/embed/h970ZBgKINg" frameborder="0" allowfullscreen="true"> </iframe>
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/cpxKbf51ccU" frameborder="0" allowfullscreen="true">
+  </iframe>
 </figure>
 
-## Infrastructura como código en EOS CR
-
-* Documentación de Openstack
+* * *
 
-https://github.com/edenia/openstack-docs
+### Introducción a HashiCorp Terraform 
 
-* Documentación de Kubernetes 
+<figure class="video_container">
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/h970ZBgKINg" frameborder="0" allowfullscreen="true">
+  </iframe>
+</figure>
 
-https://github.com/edenia/openstack-docs/blob/master/Kubernetes.md
+## Introducción a Docker y Kubernetes
 
+### Docker
 
-## Introducción a Docker y Kubernetes
+Docker es un proyecto de código abierto que automatiza el despliegue de aplicaciones dentro de contenedores de software, proporcionando una capa adicional de abstracción y automatización de virtualización de aplicaciones en múltiples sistemas operativos.
 
-### Contenedores 101
+#### **Containers 101**
 
 <figure class="video_container">
-  <iframe width="100%" height="315" src="https://www.youtube.com/embed/VqLcWftIaQI" frameborder="0" allowfullscreen="true"> </iframe>
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/VqLcWftIaQI" frameborder="0" allowfullscreen="true">
+  </iframe>
 </figure>
 
-Mira la completa introducción a la lista de reproducción de los contenedores aquí:
-https://www.youtube.com/embed/EnJ7qX9fkcU&list=PL7bmigfV0EqQt5_pBPQ8tsZjI1w68-e0H
+>[Más informacion sobre containers](https://www.youtube.com/channel/UCdkGV51Nu0unDNT58bHt9bg/search?query=containers)
```

Comment:
informacion -> información

Reply:
![image](https://user-images.githubusercontent.com/76976401/106071679-c8047280-60d4-11eb-8ad2-04916c73aade.png)
Changes were already made
Check the netlify/guias/deploy-preview — Deploy preview ready!